<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>FORM PENGGAJIAN</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>331</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="noPenggajianLabel">
      <property name="text">
       <string>No Penggajian</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="noPenggajianLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="periodeGajiLabel">
      <property name="text">
       <string>Periode Gaji</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="periodeGajiLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="tanggalLabel">
      <property name="text">
       <string>Tanggal</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDateEdit" name="tanggalDateEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="kodeKaryawanLabel">
      <property name="text">
       <string>Kode Karyawan</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="kodeKaryawanComboBox"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="gajiPokokLabel">
      <property name="text">
       <string>Gaji Pokok</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="gajiPokokLineEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="tunjTransportasiLabel">
      <property name="text">
       <string>Tunj. Transportasi</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="tunjTransportasiLineEdit"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="tunjMakanLabel">
      <property name="text">
       <string>Tunj. Makan</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="tunjMakanLineEdit"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="totalLemburLabel">
      <property name="text">
       <string>Total Lembur</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="totalLemburLineEdit"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="totalBonusLabel">
      <property name="text">
       <string>Total Bonus</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="totalBonusLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnSimpan">
   <property name="geometry">
    <rect>
     <x>304</x>
     <y>340</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Simpan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUbah">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>340</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ubah</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHapus">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>340</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
  <widget class="QTableView" name="tablePenggajian">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>400</y>
     <width>581</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCetak">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>340</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cetak</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
